Source/WebCore/page/LocalFrameView.h

```diff
@@ -735,6 +735,11 @@ class LocalFrameView final : public FrameView {
     Color scrollbarTrackColorStyle() const final;
     ScrollbarGutter scrollbarGutterStyle() const final;
     ScrollbarWidth scrollbarWidthStyle() const final;
+    void updateAnchorElement() final;
```

Comment:
In FrameView here we should refer to the "ScrollAnchoringElement", so it's easily distinguished from the element referenced by a `#` in the URL which is also called an "anchor element".

Reply:
Fixed.